src/DssSpell.t.sol

```diff
@@ -604,12 +620,44 @@ contract DssSpellTest is DssSpellTestBase {
     GemAbstract          rwagem_014            = GemAbstract(addr.addr("RWA014"));
     GemJoinAbstract      rwajoin_014           = GemJoinAbstract(addr.addr("MCD_JOIN_RWA014_A"));
     RwaUrnLike           rwaurn_014            = RwaUrnLike(addr.addr("RWA014_A_URN"));
+    RwaJarLike           rwajar_014            = RwaJarLike(addr.addr("RWA014_A_JAR"));
     RwaOutputConduitLike rwaconduitout_014     = RwaOutputConduitLike(addr.addr("RWA014_A_OUTPUT_CONDUIT"));
-    GemAbstract          psmGem                = rwaconduitout_014.gem();
+    GemAbstract          psmGem                = GemAbstract(rwaconduitout_014.gem());
     RwaInputConduitLike  rwaconduitinurn_014   = RwaInputConduitLike(addr.addr("RWA014_A_INPUT_CONDUIT_URN"));
     RwaInputConduitLike  rwaconduitinjar_014   = RwaInputConduitLike(addr.addr("RWA014_A_INPUT_CONDUIT_JAR"));
     uint256 daiPsmGemDiffDecimals              = 10 ** (dai.decimals() - psmGem.decimals());
 
+    function testRWA014_CONTRACT_DEPLOYMENT_SETUP() public {
```

Comment:
Leave a comment in the code here, saying that the pattern is to have tests in the main onboarding code